dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/UiPluginControllerTest.java

```diff
@@ -53,6 +63,8 @@ public class UiPluginControllerTest extends AbstractControllerTest {
     private static final ImmutableMap<String, Object> uiPluginServiceResult =
             ImmutableMap.of(Constants.STATUS, Status.SUCCESS, Constants.DATA_LIST, "Test Data");
 
+    private static final Logger logger = LoggerFactory.getLogger(TenantControllerTest.class);
+
```

Comment:
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(TenantControllerTest.class);

```

Reply:
> 
Okay, I have removed the code from the places you mentioned